src/App.js

```diff
@@ -29,14 +31,15 @@ class App extends Component {
   render() {
     return (
       <div>
-        <form>
-          <input
+        <h1>To Do App</h1>
+        <form className="input-div">
```

Comment:
"input-div" is not a good name because its not a div, and it doesnt say what inputs it contains. try and make classnames more descriptive